apps/web/components/ui/form/Select.tsx

```diff
@@ -1,5 +1,14 @@
 import React, { useCallback, useEffect, useState } from "react";
-import ReactSelect, { components, GroupBase, Props, InputProps, SingleValue, MultiValue } from "react-select";
+import { Check } from "react-feather";
```

Comment:
we have our own import for Icons. Look for

`import { Icon } from "@calcom/ui/Icon";`

and then 

`<Icon.FiCheck className="h-5 w-5" />`

Reply:
Done

Reply:
I still see direct import from react-feather, did you forget to commit?